src/zcash/memo.h

```diff
@@ -5,7 +5,10 @@
 #ifndef ZCASH_ZCASH_MEMO_H
 #define ZCASH_ZCASH_MEMO_H
 
+#include <array>
+#include <optional>
 #include <tl/expected.hpp>
+#include <variant>
```

Comment:
Nit: `tl/expected` is a third-party dependency, not a stdlib dependency, so we usually group them separately:
```suggestion
#include <tl/expected.hpp>

#include <array>
#include <optional>
#include <variant>
```

Reply:
Thanks. I’ll apply this suggestion after I see the Mac build finish (and hopefully succeed), then I’ll kick off the Tekton build, too.

Reply:
It was not successful, but made the requisite changes and am trying again.